subgraphs/uniswap-v3/config/_paramConfig.ts

```diff
@@ -0,0 +1,197 @@
+import { dataSource, BigDecimal, log } from "@graphprotocol/graph-ts";
+import { Network } from "../src/common/constants";
+import { toLowerCase } from "../src/common/utils/utils";
+
+export namespace Protocol {
+  export const UNISWAPV3 = "Uniswap V3";
+}
+
+// Choose which protocol you are indexing. The deployed network will already be determined
+let PROTOCOL_NAME_TEMP = Protocol.UNISWAPV3;
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Divider should be less than or equal to 80 characters per row for readability

Reply:
Split to 2 rows